server/re/src/icatGeneralMS.cpp

```diff
@@ -141,7 +147,7 @@ int  msiSetResource( msParam_t* xrescName, ruleExecInfo_t *rei ) {
     char* rescName{static_cast<char *>(xrescName->inOutStruct)};
 
     if (rei->doi == nullptr) {
-        rodsLog(LOG_ERROR, "Cannot set resource, DOI not found.");
+        log_msi::error("Cannot set resource, DOI not found.");
```

Comment:
`DOI` -> `DOI (DataObjInfo)`

Reply:
Should we just make this in plain english, "Data Object Info"?

Reply:
No. `DataObjInfo` is a real thing in iRODS and using plain english to try and refer to it would only lead to more confusion.

Reply:
Done.